static/builder/dataStorage.js

```diff
@@ -1,3 +1,6 @@
+// Constants
+const DATA_STORAGE_EASILY_OBTAINABLE_ITEMS = ["shop", "chest", "quest", "key", "colosseum", "recipe", "recipe-chest", "recipe-key", "recipe-quest"];
```

Comment:
You're probably missing recipe-shop and recipe-colosseum. I also thing that "recipe" alone shouldn't exist, but I put it in the data, so it can stay I guess ;-)

Reply:
Nice catch, adding those! I tried going over the data, but didn't find those myself.